shell/platform/windows/compositor_opengl.cc

```diff
@@ -92,14 +92,9 @@ bool CompositorOpenGL::CollectBackingStore(const FlutterBackingStore* store) {
   return true;
 }
 
-bool CompositorOpenGL::Present(FlutterViewId view_id,
+bool CompositorOpenGL::Present(FlutterWindowsView* view,
                                const FlutterLayer** layers,
                                size_t layers_count) {
```

Comment:
Depending on how paranoid you feel, consider tossing in an `FML_DCHECK(view != nullptr)` in the updated methods to shout should anyone ever violate preconditions.

Reply:
Added!